docs/concepts/state/life-cycle.md

```diff
@@ -87,22 +88,20 @@ After creating the state by calling its constructor, NGXS calls the lifecycle ho
 | ngxsOnInit()         | Called _once_, after the _first_ `ngxsOnChanges()` and _before_ the `APP_INITIALIZER` token is resolved. |
 | ngxsAfterBootstrap() | Called _once_, after the root view and all its children have been rendered.                              |
 
-## Feature Modules Order of Imports
+## Feature States Order of Imports
 
-If you have feature modules they need to be imported after the root module:
+If you have feature states they need to be registered after the root `provideStore` has been called:
 
 ```ts
-// feature.module.ts
-@NgModule({
-  imports: [NgxsModule.forFeature([FeatureState])]
-})
-export class FeatureModule {}
+// some-data-access-library/index.ts
+export function provideDataAccessInvoiceLines() {
+  return provideStates([InvoiceLinesState]);
```

Comment:
We have removed the example with the `NgxsModule.forFeature` usage.
We should possibly still maintain the Module version of the example (potentially as a collapsible section within the page?)